.changeset/angry-teachers-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'hasura-auth': patch
+---
+
+Fix user roles aggregation. Only one role was returned by the SQL query.
```

Comment:
there's no need to add a changeset as this bug was introduced by code that is not yet published. You can remove the file

Reply:
done